sql/expression/function/sleep.go

```diff
@@ -0,0 +1,68 @@
+package function
+
+import (
+	"fmt"
+	"time"
+
+	"gopkg.in/src-d/go-mysql-server.v0/sql"
+	"gopkg.in/src-d/go-mysql-server.v0/sql/expression"
+)
+
+// Sleep is a function that just waits for the specified number of seconds
+// and returns 0.
+// It can be useful to test timeouts or long queries.
+type Sleep struct {
+	expression.UnaryExpression
+}
+
+// NewSleep creates a new Sleep expression.
+func NewSleep(e sql.Expression) sql.Expression {
+	return &Sleep{expression.UnaryExpression{Child: e}}
+}
+
+// Eval implements the Expression interface.
+func (s *Sleep) Eval(ctx *sql.Context, row sql.Row) (interface{}, error) {
+	child, err := s.Child.Eval(ctx, row)
+
+	if err != nil {
+		return nil, err
+	}
+
+	if child == nil {
+		return nil, nil
+	}
+
+	child, err = sql.Int32.Convert(child)
+	if err != nil {
+		return nil, err
+	}
+
+	if child.(int32) > 0 {
+		time.Sleep(time.Duration(child.(int32)) * time.Second)
```

Comment:
Based on mysql docs a fraction can be used, so my understanding is that someone can write `sleep(0.5)` to sleep for 500 ms. So, I think it's better to make it a float function.

Reply:
True! I'll fix it in the next push.

Reply:
Fixed and pushed, PTAL when you can.